vanilla-framework/vanilla-framework
# Baseline fixes

## Done

Due to browser rounding, text elements start to drift from the baseline grid after a few hundred lines.
To minimise this, I'm adding a small constant that forces the rounding to happen so that text stays on the baseline. 

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/
- Paste 1000 lines of text in any heading style (or checkout this [file ](https://github.com/lyubomir-popov/vanilla-framework/blob/thousand-line-test/examples/templates/typographic-spacing.html) that includes a script to generate text)
- Observe no drift from baseline

## Details

[List of links to issues/bugs and cards if needed]

## Screenshots

[if relevant, include a screenshot]
